plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -48,12 +47,17 @@ def __init__(self, configurations: Dict[str, Dict], use_external_lib: bool = Fal
         self._check_config_equal(
             key="peft.quantization.auto_gptq.from_quantized", value=True
         )
-        self.use_external_lib = use_external_lib
+        self.use_external_lib = self._check_config_and_maybe_check_values(
```

Comment:
doing this will force a check that the config must have `external_lib`

Im not sure if `self._check_config_and_maybe_check_values(..., values=[True, False, None]` implements optional behavior. If this doesnt work, then maybe you can implement a `default` behavior

```
self._check_config_and_maybe_check_values(..., values=[True, False], default=False)
```

means that if the key is missing, it will set to False.